package.json

```diff
@@ -5,47 +5,58 @@
   "keywords": [
     "ember-addon"
   ],
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/rwjblue/ember-router-helpers.git"
+  },
   "license": "MIT",
   "author": "Robert Jackson <me@rwjblue.com>",
   "directories": {
     "test": "tests"
   },
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/rwjblue/ember-router-helpers.git"
-  },
   "scripts": {
     "build": "ember build",
-    "start": "ember server",
-    "test": "ember try:each"
+    "lint:js": "eslint .",
+    "start": "yarn ember serve",
+    "test": "yarn ember test",
+    "test:all": "ember try:each",
+    "ember": "./node_modules/.bin/ember",
+    "ember:update": "./node_modules/.bin/ember-cli-update"
   },
   "dependencies": {
-    "ember-cli-babel": "^6.0.0",
+    "ember-cli-babel": "^6.6.0",
     "ember-cli-htmlbars": "^2.0.2",
     "ember-router-service-polyfill": "^1.0.1"
   },
   "devDependencies": {
-    "broccoli-asset-rev": "^2.4.5",
+    "broccoli-asset-rev": "^3.0.0",
     "ember-ajax": "^3.0.0",
-    "ember-cli": "~2.14.0-beta.2",
+    "ember-cli": "~3.3.0",
     "ember-cli-dependency-checker": "^2.0.0",
-    "ember-cli-eslint": "^4.0.0",
-    "ember-cli-htmlbars-inline-precompile": "^0.4.3",
+    "ember-cli-eslint": "^4.2.1",
+    "ember-cli-htmlbars-inline-precompile": "^1.0.0",
     "ember-cli-inject-live-reload": "^1.4.1",
-    "ember-cli-qunit": "^4.0.0",
-    "ember-cli-shims": "^1.2.0-beta.2",
+    "ember-cli-qunit": "^4.3.2",
+    "ember-cli-shims": "^1.2.0",
     "ember-cli-sri": "^2.1.0",
-    "ember-cli-uglify": "^1.2.0",
+    "ember-cli-uglify": "^2.0.0",
+    "ember-cli-update": "^0.23.0",
```

Comment:
please remove, ember-cli-update should only be installed globally IMO

Reply:
aight. personally, _because_ I have so many projects with different versions of things, I try to use local bins as much as possible

Reply:
Generally I feel the same way, but with ember-cli-update you _never_ want an older version

Whenever it is used it should be installed again (to get all the bug fixes, new codemods, etc)...

Reply:
does ember-cli-update check itself for updates?
(maybe that's something I could submit a PR for if it doesn't already exist)

Reply:
I can't recall, but I think that was recently added....